articles/install-vpp-apps-on-macos-using-fleet.md

```diff
@@ -0,0 +1,138 @@
+
+
+### Add an app to Fleet
+
+1. **Navigate to the Software page**: Click on the "Software" tab in the main navigation menu.
+
+2. **Select your team**: Click on the "All teams" dropdown in the top left of the page and select
+
+   your desired team.
+
+3. **Open the "Add software" modal**: Click on the "Add software" button in the top right of the
+
+   page.
+
+4. **View your available apps**: Click on the "App Store (VPP)" tab in the "Add software" modal. The modal will list the apps that you have purchased through VPP but still need to add to Fleet.
+
+5. **Add an app**: Select an app from the list and click the "Add software" button in the bottom
+
+   right of the modal. The app should appear in the software list for the selected team.
+
+    
+
+
+### Remove an app from Fleet
+
+1. **Navigate to the Software page**: Click "Software" in the main navigation menu.
+
+2. **Find the app you want to remove**: Search for the app using the search bar in the top right corner of the table.
+
+3. **Access the app's details page**: Click on the app's name in the table.
+
+4. **Remove the app**: Click on the "Actions" dropdown on the right side of the page. Click
+
+   "Delete," then click "Delete" on the confirmation modal. Deleting an app will not
+
+   uninstall the app from the hosts on which it was previously installed.
+
+
+### Installing apps on macOS hosts
+
+1. **Add a macOS host to the relevant team.**
+
+2. **Go to the host's detail page**: Click the "Hosts" tab in the main navigation menu. Filter the
+
+   hosts by the team, and click the host's name to see its details page.
+
+3. **Find the app**: Click the "Software" tab on the host details page. Search for
+
+   the software you added in the software table's search bar.
+
+4. **Install the app**: Click the "Actions" dropdown on the far right of the app's entry
+
+   in the table. Click "Install" to trigger an install. This action will send an MDM command to the
+
+   host instructing it to install the app.
+
+    
+
+
+## Managing apps with `fleetctl`
+
+Fleet’s command line tool, `fleetctl`, will allow you to manage your organization’s apps via the `fleetctl gitops` command. You can add apps to your team’s configuration YAML file like so:
+
+```yaml
+
+name: "Team VPP"
+software:
+  app_store_apps:
+	- app_store_id: "12345678"
+# other config....
+
+```
+
+and then run the `fleetctl gitops apply` command to apply the configuration.
+
+> To obtain the App Store ID, search for the app you want to add on the App Store website. The ID of the app is in its URL, which is formatted like so: `http://apps.apple.com/<country>/app/<app–name>/id<store-ID>`
+
+Learn more about [`fleetctl` CLI](https://fleetdm.com/docs/using-fleet/fleetctl-cli) and the [gitops workflow](https://fleetdm.com/docs/using-fleet/gitops).
```

Comment:
```suggestion
To manage App Store apps using Fleet's best practice GitOps, check out the `software` key in the GitOps reference documentation [here](https://fleetdm.com/docs/using-fleet/gitops#software).
```

I think we can keep this short and link out to GitOps reference docs. That way, we reduce duplicate docs (YAML example) and help move folks to the reference so they can see the full GitOps workflow.